src/stcal/ramp_fitting/ramp_fit_class.py

```diff
@@ -41,7 +42,7 @@ def __init__(self):
 
         self.current_integ = -1
 
-    def set_arrays(self, data, err, groupdq, pixeldq):
+    def set_arrays(self, data, err, groupdq, pixeldq, average_dark_current):
```

Comment:
New parameter needs to be added to docstring.

Reply:
Added ✅